fml/raster_thread_merger.cc

```diff
@@ -133,7 +132,7 @@ bool RasterThreadMerger::IsEnabled() {
 }
 
 bool RasterThreadMerger::IsEnabledUnSafe() const {
-  return enabled_;
+  return shared_merger_->IsEnabledUnSafe();
```

Comment:
It's a bit odd that the writing of this is in a mutex lock, where reading is not.

Reply:
@gaaclarke Thanks! 💪 I also have some changes about the thread merger, I will deal with it together with another PR.